.github/issue-template.md

```diff
@@ -4,9 +4,7 @@
 
 ## Steps to Reproduce the Problem
 
-1.
-2.
-3.
+1. 2. 3.
```

Comment:
@evankanderson looks like we need to address some things before this is completely clean.